submissions/chri911/popup-menu/style.css

```diff
@@ -0,0 +1,173 @@
+html {
+    box-sizing: border-box;
+}
+*,
+*::before,
+*::after{
+    box-sizing: inherit;
+}
+* {
+	margin: 0;
+	padding: 0;
+}
+a{
+    text-decoration: none;
+    
+}
+.container{
+    display: flex;
+    justify-content: flex-end;
+    padding: 10px 15px;
+    align-items: center;
+    color: #5f5c5f;
+    }
+.container > div{
+        margin-right: 20px;
+    }
+#popup,
+.notes,
+.user{
+    width: 30px;
+    height: 30px;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+.notes{
+    background-image: url(images/bell.png);
+}    
+.user{
+    background-image: url(images/photo.jpg);
+    border-radius: 50%;
+} 
+#popup{
+    background-image: url(images/apps.jpg);
+    display: block;
+}
+.popup1{
+    position: relative;
+    height: 100%;
+}
+#isexpanded, .popup__menu, #expanded{
+    display: none;
```

Comment:
Watch how to hide checkbox in right way. [Video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s)